### PR TITLE
Excluding node_modules folder as Rails 6 start to including node packages 

### PR DESCRIPTION
**Pull request by kelvin8773**
As [Rails 6](https://weblog.rubyonrails.org/releases/) start to integrate javascript by default and put "yarn" as packages management tools.

So it is better excluding "node_modules" for rubocop linter test", I have encountered some issues when trying to integrate bootstrap & Fontawesome node packages into the project. 

I hope it helps others.

### CODE REVIEW

File: ruby/.rubocop.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ AllCops:
     - "config/**/*"
     - "Guardfile"
     - "Rakefile"
+    - "node_modules/**/*"
```
Comment (nidalaa) [inline ruby/.rubocop.yml]: Hi @kelvin8773 !

Thank you for this contribution! 🙇 
Could you also add this dir here to ignored files in Stickler config: https://github.com/microverseinc/linters-config/blob/master/ruby/.stickler.yml#L5 ?

Comment (kelvin8773) [reply ruby/.rubocop.yml]: ok, updated, please check!

File: ruby/.stickler.yml [hunk 2]
```diff
@@ -2,19 +2,14 @@
...
 files:
   ignore:
     - "bin/*"
     - "db/*"
     - "config/*"
     - "Guardfile"
     - "Rakefile"
+    - "node_modules/**/*"
 
-# PLEASE DO NOT enable auto fixing options
```
Comment (nidalaa) [inline ruby/.stickler.yml]: @kelvin8773 well done but something went wrong during merging last changes from master :)
Can you make sure that extra comments added before are not going to disappear with your change? 🙏

Comment (kelvin8773) [reply ruby/.stickler.yml]: Sorry for that, I guess it is because my local repo is outdate, now should be fine now, please check again!

Kelvin

Comment (nidalaa) [reply ruby/.stickler.yml]: @kelvin8773 you are still removing these lines :)

To make it more clear, I mean this text:

```
# PLEASE DO NOT enable auto fixing options
# if you need extra support from you linter - do it in your local env as described in README for this config
# find full documentation here: https://stickler-ci.com/docs
```

**Comment by kelvin8773** [comment]
Fixed, I hope there is no more mistakes. : p

**Comment by nidalaa** [comment]
Hi @kelvin8773 !

We are ready to :shipit: 

Thanks you!

**Pull request merged**